pkg/query-service/app/preferences/bootstrap_preferences.json

```diff
@@ -0,0 +1,38 @@
+[
+    {
+        "id": "LOGS_SAVED_VIEWS",
+        "name": "Logs saved views",
+        "default_value":"false",
+        "depends_on":"",
+        "user": 1,
+        "org": 1,
+        "group_id": "SAVED_VIEWS"
+    },
+    {
+        "id": "LOGS_TOOLBAR_COLLAPSED",
+        "name": "Logs toolbar collapsed",
+        "default_value":"false",
+        "depends_on":"",
```

Comment:
these are dummy data and will be removed

Reply:
Please include all the features that support today in the bootstrap set.

Reply:
inline comment should work ?

Reply:
removed the dummy preferences from here as per the discussion

Reply:
removed the dummy preferences from here as per the discussion